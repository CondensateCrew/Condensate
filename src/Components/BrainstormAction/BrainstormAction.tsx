import React from 'react';

const BrainstormAction:React.FC = () => {
  return (
    <div>
      <p>This is a suggested action: 'Create an App'</p>
    </div>
  )
}
