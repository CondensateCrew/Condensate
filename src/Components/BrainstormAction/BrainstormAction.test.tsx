import React from 'react';
import { shallow, mount } from 'enzyme';
import BrainstormAction from './BrainstormAction';

describe('BrainstormAction', () => {
  let wrapper;
  beforeEach(() => {
    interface Props {
      action: string,
      key: number
    }
  });

  afterEach(() => {
    jest.clearAllMocks();
  });

  it('should match the snapshot', () => {
    const mockAction = 'create an app';
    const mockKey = 4;

    wrapper = shallow(<BrainstormAction 
      action={mockAction}
      key={mockKey}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});