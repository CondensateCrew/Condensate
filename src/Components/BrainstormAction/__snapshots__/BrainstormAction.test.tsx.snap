// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrainstormAction should match the snapshot 1`] = `
<option>
  create an app
</option>
`;
